src/rust/ide/src/controller/module.rs

```diff
@@ -170,6 +203,23 @@ shared! { Handle
         pub fn subscribe_graph_notifications(&mut self) -> Subscriber<notification::Graphs> {
             self.graph_notifications.subscribe()
         }
+
+        /// Returns metadata for given node, if present.
+        pub fn node_metadata(&mut self, id:ast::ID) -> FallibleResult<NodeMetadata> {
+            self.module.metadata.ide.node.get(&id).cloned()
+                .ok_or_else(|| NodeMetadataNotFound(id).into())
```

Comment:
According to our guide we should avoid to make mutliline expressions, rather create aliases for each step for such chain.
```suggestion
            let metadata = self.module.metadata.ide.node.get(&id).cloned();
            metadata.ok_or_else(|| NodeMetadataNotFound(id).into())
```